services/types.go

```diff
@@ -76,6 +75,61 @@ var (
 	}
 )
 
+// Types and wrappers for types that are not specific to one service
+type StableToken struct {
```

Comment:
this is much cleaner

Reply:
thanks for the helpful discussion on this earlier!